### PR TITLE
WIP: Config parser

### PR DESCRIPTION
**Pull request by daanbreur**
This PR contains commits that make functions

- importVafConfig `public`:
  -  return type *VafConfigStructure*
- parseWordlist `private`:
  - parameter type *JsonNode*
  - returns type *VafWordlist*
- parseVafConfig `public`:
  - parameter type *JsonNode*
  - returns type *VafConfigStructure*

### CODE REVIEW

**Comment by daanbreur** [comment]
This pr is in relation with #7 and #8

File: src/config/importVafConfig.nim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import json
+import os
+import strutils
+
+import initVafConfig
+import parseVafConfig
+
+import VafConfigStructure
+
+var homeDir: string = os.getHomeDir()
```
Comment (d4rckh) [inline src/config/importVafConfig.nim:10]: maybe put this in a separate file? it gets quite repetitive

File: src/config/importVafConfig.nim [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+proc importVafConfig*(): VafConfigStructure = 
+    var configExists: bool = fileExists(homedir & "/.vaf.json")
```
Comment (d4rckh) [inline src/config/importVafConfig.nim:13]: this is also quite repetitive

File: src/config/importVafConfig.nim (hunk 1 above)
Comment (d4rckh) [inline src/config/importVafConfig.nim:10]: maybe put this in a separate file? it gets quite repetitive

File: src/config/importVafConfig.nim (hunk 2 above)
Comment (daanbreur) [reply src/config/importVafConfig.nim:13]: What is repetitive? The filepath?

File: src/config/importVafConfig.nim (hunk 1 above)
Comment (daanbreur) [reply src/config/importVafConfig.nim:10]: What should we put in a seperate file?

File: src/config/importVafConfig.nim (hunk 2 above)
Comment (d4rckh) [reply src/config/importVafConfig.nim:13]: > What is repetitive? The filepath?

that and "configExists" is in almost every config file

File: src/config/importVafConfig.nim (hunk 1 above)
Comment (d4rckh) [reply src/config/importVafConfig.nim:10]: > What should we put in a seperate file?

not sure, maybe a constsnts.nim in src/

Comment (daanbreur) [reply src/config/importVafConfig.nim:10]: Good Idea! 👍 Lets refactor later tho

**Pull request merged**